index.php

```diff
@@ -9,4 +9,135 @@
    Author URI: https://mxstbr.com
    License: MIT
    */
+
+// Inject the feedback.fish JavaScript file into the DOM just before the body
+function enqueue_feedback_fish_script()
+{
+    $projectId = get_option('feedback_fish_project_id');
+    wp_register_script(
+        'feedback-fish',
+        "https://feedback.fish/ff.js?pid=$projectId",
+        [],
+        null,
+        true
+    );
+    wp_enqueue_script('feedback-fish');
+}
+add_action('wp_enqueue_scripts', 'enqueue_feedback_fish_script');
+
+// Add defer="defer" to the feedback.fish <script> tag to avoid blocking render
+// Taken from: https://wordpress.stackexchange.com/a/38335
+add_filter(
+    'script_loader_tag',
+    function ($tag, $handle) {
+        if ('feedback-fish' !== $handle) {
+            return $tag;
+        }
+
+        return str_replace(' src', ' defer="defer" src', $tag);
+    },
+    10,
+    2
+);
+
+// Add "Send feedback" to the primary nav if it exists
+function add_feedback_fish_nav_menu_item($items, $args)
+{
+    $current_user = wp_get_current_user();
+    $manual_usage = get_option('feedback_fish_manual');
+    if (!$manual_usage) {
+        $items .= "<li class=\"menu-item\"><a href=\"#\" data-feedback-fish-userid=\"$current_user->user_email\" data-feedback-fish>Send feedback</a></li>";
+    }
+    return $items;
+}
+add_filter('wp_nav_menu_items', 'add_feedback_fish_nav_menu_item', 10, 2);
+
+// Add "Feedback Fish" settings page to the WPAdmin menu
+function create_feedback_fish_settings_page()
+{
+    $page_title = 'Feedback Fish Settings';
```

Comment:
For strings that are rendered in admin UIs, I'd recommend getting in the habit of passing them through translation functions so that in the future you could provide translations for the strings in your plugin and make it useful for many languages. 


see: https://codex.wordpress.org/I18n_for_WordPress_Developers#Strings_for_Translation

So, in your case it would become: `$menu_title = __( 'Feedback Fish', 'feedback-fish' );`

The 2nd argument is your plugin's "text domain" which you define in your plugin docblock like so: https://codex.wordpress.org/I18n_for_WordPress_Developers#Text_Domains